examples/user_interfaces/svg_tooltip.py

```diff
@@ -1,3 +1,6 @@
+#! /usr/bin/env python
```

Comment:
It's not executable, so I don't think we need this (and I don't think it should be made executable either.)